src/config/configuration.js

```diff
@@ -0,0 +1,16 @@
+import { config } from 'dotenv';
+config();
+
+const envVars  = process.env;
+
+
+const configuration = Object.freeze({
+
+port : envVars.PORT,
+
+}); 
+
+
```

Comment:
Please remove extra space and console.log

Reply:
Done